### ISSUE TITLE
Metrics aren't properly formatted by locale

### ISSUE THREAD

**Issue by marijoo** [laravel/nova-issues#2869]
- Laravel Version: 8.3.0
- Nova Version: 3.10.0
- PHP Version: 7.4.x

### Description:
Nova metrics don't respect locale specific formats  because the numbro languages aren't imported anymore.

See #1472

### Fix:
The `numbro.setLanguage()` call based on `window.config.locale` is already there. So importing the languages will fix the issue:

```js
// resources/js/util/numbro.js
import numbro from 'numbro'

// add these two lines
import numbroLanguages from 'numbro/dist/languages.min'
Object.values(numbroLanguages).forEach(language => numbro.registerLanguage(language))

if (window.config.locale) {
  numbro.setLanguage(window.config.locale.replace('_', '-'))
}

...
```

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.

**Comment by marijoo** [comment]
This is a bug, not a feature request, imho. 😒

**Comment by zedix** [comment]
@davidhemphill I agree with @marijoo  this should be considered as a bug.

A typical use case is to display a `Laravel\Nova\Metrics\Value` as a currency.

<img width="217" alt="image" src="https://user-images.githubusercontent.com/27975/99876655-7503d000-2bf8-11eb-8c9f-15d0d1c0701d.png">

But the expected result for EUR shoud be for example **12 010,00 €**

We can currently pass an `output: 'currency'` option to `numbro` but the numbro `language` do use the configurated nova locale.

```php
public function calculate(Request $request)
{
    return $this
        ->sum($request, SomeModel::where('xxx', 'xxx), 'total_price')
        ->format([
            'output' => 'currency',
            'spaceSeparated' => true,
            'mantissa' => 2,
        ]);
}
```

I would be really useful to fix this bug and configure numbro language appropriately.

The default expectation is that the Nova _locale_ is distributed and used everywhere in the front-end.

**Comment by desaintflorent** [comment]
I'm also facing this issue. Anyone found a temporary solution ?

**Comment by crynobone** [comment]
This has been implemented on v3.20.0

**Comment by zedix** [comment]
@crynobone Thanks ! 

Just tried your fix with v3.20 and noticed that using `app.locale => 'fr'` **doesn't work**, while using `fr_FR`' is OK.

I'm using the above format options inside calculate():

```
->format([
    'output' => 'currency',
    'thousandSeparated' => true,
    'mantissa' => 0,
]);
```

Result with locale => `fr`
<img width="168" alt="image" src="https://user-images.githubusercontent.com/27975/105608759-2cf25c80-5da5-11eb-9d76-a0cf05e0bc37.png">

Result with locale => `fr_FR`
<img width="164" alt="image" src="https://user-images.githubusercontent.com/27975/105608660-9625a000-5da4-11eb-9402-af1293f0fab1.png">

Inside utils/numbro.js, you probably want to transform any 2-letters `window.config.locale` to its long version (`${locale}-${locale.toUpperCase()}`) before calling `numbro.registerLanguage(language)`

**Comment by crynobone** [comment]
Not all locale strong follow that format.

**Comment by desaintflorent** [comment]
My app.locale was set to `fr` too and not working with this fix. 
Any downside on Laravel to set it to `fr_FR` ? 
And is it possible to set the locale only for Nova ? I tried `nova.locale => 'fr_FR` without success.

**Comment by crynobone** [comment]
> Any downside on Laravel to set it to `fr_FR` ?

This should be fine, but I do think you need to update the locale files and paths to `fr_FR` from `fr`.

> And is it possible to set the locale only for Nova ? I tried `nova.locale => 'fr_FR'` without success.

The configuration isn't available 

However, you could do the following (under `App\Providers\NovaServiceProvider`):

```php
    /**
     * Bootstrap any application services.
     *
     * @return void
     */
    public function boot()
    {
        parent::boot();

        Nova::booted(function () {
            $availableLocales = ['fr' => 'fr_FR'];
            $appLocale = app()->getLocale();

            if (array_key_exists($appLocale, $availableLocales)) {
                app()->setLocale($availableLocales[$appLocale]);
            }
        });
    }
```

**Comment by NormanHuth** [comment]
> This has been implemented on v3.20.0

Yes. But how do you apply it now?
